storage/Makefile

```diff
@@ -24,7 +24,7 @@ vet:
 
 .PHONY: staticcheck
 staticcheck:
-	@go install honnef.co/go/tools/cmd/staticcheck@v0.3.3
+	@go install honnef.co/go/tools/cmd/staticcheck@v0.4.3
```

Comment:
Do we need this upgrade? It seems to have broken the build for older versions of Go:
```
# honnef.co/go/tools/go/loader
../../go/pkg/mod/honnef.co/go/tools@v0.4.3/go/loader/loader.go:233:4: unknown field 'Instances' in struct literal of type types.Info
note: module requires Go 1.19
```

Reply:
Oh dang.. the old version doesn't work on 1.20.  No worries, will revert.

Reply:
```suggestion
	@go install honnef.co/go/tools/cmd/staticcheck@v0.3.3
```